src/Components/Users/users.jsx

```diff
@@ -1,44 +1,79 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
 import React from 'react';
+import * as axios from 'axios';
 
 import { I18N } from '../../constants';
-import usersArray from '../../usersArray';
 
 import style from './users.module.css';
 
-const users = (props) => {
-  if (props.users.length === 0) {
-    props.setUsers(usersArray);
+import defaultImg from '../Image/person.jpg';
+
+class Users extends React.Component {
+  componentDidMount() {
+    axios.get(`https://social-network.samuraijs.com/api/1.0/users?page=${this.props.currentPage}&count=${this.props.pageSize}`)
+      .then((response) => {
+        this.props.setUsers(response.data.items);
+        this.props.setTotalUsersCount(response.data.totalCount);
+      });
   }
-  const toggleButton = (followed, id) => (followed ? <button type="button" onClick={() => { props.unfollowUser(id); }}>{I18N.ENG.UNFOLLOW}</button>
-    : <button type="button" onClick={() => { props.followUser(id); }}>{I18N.ENG.FOLLOW}</button>);
 
-  const usersItem = ({
-    id, photoUrl, followed, fullname, status, location,
-  }) => (
-    <div key={id} className={style.item}>
+  render() {
```

Comment:
I think we need to have a discussion about the things you put into "render" method.

Reply:
it was very useful discussion